src/components/Modals/RemoveLiquidity/index.js

```diff
@@ -46,14 +47,14 @@ function RemoveLiquidityModal({
   const decimals1 = getDecimalsForAsset(asset1, assetData1?.decimals);
   const decimals2 = getDecimalsForAsset(asset2, assetData2?.decimals);
 
-  const calculateAssetToBurn = (numberValue) => {
-    const tokensToBurn = new BN(lpTokensBalance).mul(new BN(numberValue)).div(new BN(BN_HUNDRED));
+  const calculateAssetToBurn = async (numberValue) => {
+    const tokensToBurn = new BN(lpTokensBalance).mul(new BN(numberValue)).div(BN_HUNDRED);
     setTokensToBurnState(tokensToBurn);
-    const fee = 1; // 0.1%
+    const fee = new BN(withdrawlFee);
     const calculatedAmount1 = calculatePooled(tokensToBurn, liquidity, reserved.asset1);
     const calculatedAmount2 = calculatePooled(tokensToBurn, liquidity, reserved.asset2);
-    const asset1Data = calculatedAmount1.mul(new BN(BN_THOUSAND - fee)).div(new BN(BN_THOUSAND));
-    const asset2Data = calculatedAmount2.mul(new BN(BN_THOUSAND - fee)).div(new BN(BN_THOUSAND));
+    const asset1Data = !fee.isZero() ? calculatedAmount1.sub(calculatedAmount1.div(fee)) : calculatedAmount1;
+    const asset2Data = !fee.isZero() ? calculatedAmount2.sub(calculatedAmount2.div(fee)) : calculatedAmount2;
```

Comment:
Why `amount - amount/fee`? Shouldn't it be `amount * (1000-fee)/1000` or `amount - amount * fee/1000`?